nmostesting/suites/IS1101Test.py

```diff
@@ -1502,11 +1544,12 @@ def test_03_01(self, test):
             if len(self.outputs) == 0:
                 return test.UNCLEAR("No outputs")
             return test.PASS()
-        return test.FAIL("The request has not succeeded: {}".format(response.json()))
+        return test.FAIL("The outputs streamcompatibility request has failed: {}"
+                         .format(response.json()))
 
     def test_03_02(self, test):
         """
-        Verify that some of the outputs of the device are connected.
+        The receivers verify that some of the outputs of the device are connected.
```

Comment:
@gwgeorgea, @alabou, does this test case check that the receivers verify outputs are connected, or does this test case check/verify that outputs are connected? I assumed from reading the code that the words "The receivers" had been introduced erroneously last time?